Softeq.NetKit.Chat.Domain/DomainModels/Message.cs

```diff
@@ -17,6 +17,7 @@ public class Message : IBaseEntity<Guid>, ICreated
         public DateTimeOffset Created { get; set; }
         public DateTimeOffset? Updated { get; set; }
         public MessageType Type { get; set; }
+        public MessageAccessibility IsArchived { get; set; }
```

Comment:
It's not good name of the filed. We may have more than two statuses. It would be better to rename to something like 'AccessibilityStatus'.

Reply:
Done